interface/ceed.c

```diff
@@ -301,7 +309,6 @@ int CeedFree(void *p) {
   @ref Backend
 **/
 int CeedRegister(const char *prefix, int (*init)(const char *, Ceed), unsigned int priority) {
-  CeedDebugEnv("Backend Register: %s", prefix);
```

Comment:
Why was this dropped?

Reply:
Just moved it into `CeedRegisterImpl` so that the debugging statement would accompany the actual implementation. But I see now that the idea was to differentiate "Backend Register" and "Weak Register" print statements now, so I will revert it if that is OK with you?

Reply:
Thanks!